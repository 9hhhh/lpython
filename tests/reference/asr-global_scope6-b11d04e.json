{
    "basename": "asr-global_scope6-b11d04e",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/global_scope6.f90",
    "infile_hash": "3f2dfa122b78175565b7baa8f4d0417cfb0ed5bbe26bbb5f0f1f6f2e",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-global_scope6-b11d04e.stdout",
    "stdout_hash": "ab4c1f4c7f9e11a74c6097e350cc41b1d22c450cc0a39358e96437fc",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}