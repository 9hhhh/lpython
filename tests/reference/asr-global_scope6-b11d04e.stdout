(TranslationUnit (SymbolTable 6cabfe67 {x: (Variable 6cabfe67 x 0 (Integer 4 []))}) [(= (Var 6cabfe67 x) (Num 6 (Integer 4 []))) (BinOp (Num 2 (Integer 4 [])) operatorType2 (Var 6cabfe67 x) (Integer 4 [])) (= (Var 6cabfe67 x) (BinOp (Var 6cabfe67 x) operatorType0 (Num 1 (Integer 4 [])) (Integer 4 [])))])
